packages/verkle/src/verkleTree.ts

```diff
@@ -162,7 +162,9 @@ export class VerkleTree {
    * @returns A Promise that resolves to `Uint8Array` if a value was found or `null` if no value was found.
    */
   async get(key: Uint8Array): Promise<Uint8Array | null> {
-    verifyKeyLength(key)
+    // TODO: Determine if we actually need to do this check.  The key for an internal node will necessarily
+    // be less than 32 bytes since it's the path to that node in the trie
+    // verifyKeyLength(key)
```

Comment:
Wouldn't `get` always be used for full 32-byte keys? The way I'm thinking of it is:
- `trie.get` is the high-level method that we use for 32-byte keys (just like we `put` values using 32-byte keys)
- Get only returns actual "end values"
- The "path to the node in the trie" which is referred to in the comment wouldn't be directly accessed with the `get` method (even though the logic would navigate through these internal node swhen walking the trie) 

There I think we should verifyKeyLength

Reply:
So there's never an instance where you might want to `get` an internal node?

Reply:
I think there are instances where we'd want to retrieve an internal node, but not using the trie.get method

Reply:
Sounds good.  Will revert this change.